### PR TITLE
chore(technical-base): refactor technical base

### PR DESCRIPTION
**Pull request by gautier-lefebvre**
Still missing:
- [x] i18n workflow.
- [x] There is a bug with react-intl, it does not seem to find the current locale messages, haven't found why yet.

Implements #22.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -5,52 +5,57 @@
...
+              "chrome 69"
+            ]
           }
         ],
-        "@babel/preset-stage-1",
         "@babel/preset-react"
+      ],
+      "plugins": [
+        "@babel/plugin-proposal-class-properties",
+        "@babel/plugin-syntax-dynamic-import"
```
Comment (sylvainar) [inline .babelrc]: optionnal chaining ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,9 +58,10 @@ You will need to copy `.env.sample` to a `.env` file and configure this keys :
 
 |Property|Description|Default|
 | ------------- | ------------- | ------------- |
-|SRC_PATH|The path to the renderer root|src/renderer|
 |CI|Wheter the current environment is a CI environment|false|
 |GH_TOKEN|A GitHub token to deploy the application (see deploying)||
+|ENV_PREFIX|A prefix for environment variables specific to the app|VAULT_|
+|VAULT_WDS_PORT|webpack-dev-server port|8080|
 
 (TODO: Sentry, etc...)
```
Comment (sylvainar) [inline CONTRIBUTING.md]: hello

**Review by sylvainar — DISMISSED** [review]

File: .babelrc (hunk 1 above)
Comment (gautier-lefebvre) [reply .babelrc]: added

File: CONTRIBUTING.md (hunk 2 above)
Comment (gautier-lefebvre) [reply CONTRIBUTING.md]: added sentry dsn description

**Pull request merged**